### PR TITLE
improve pclasses to fix class stats don't change on equip/unequip/destroy armor

### PR DESCRIPTION
**Pull request by crabman77**
player class stats are updated only on switch class or join server, if hunter have reinforced armor at the time of switch he win 10 stamina, but if he unequip this armor after he don't loose 10 stamina.
this fix add ability to update stats on armor change or others.

### CODE REVIEW

File: mods/mff/mff_pclasses/wizard.lua [hunk 1]
```diff
@@ -13,12 +13,18 @@ pclasses.api.register_class("wizard", {
...
 		-- Add specs here
-		mana.setmax(pname, mana.getmax(pname)+100)
+		pclasses.api.util.on_update(pname)
 		minetest.log("action", "[PClasses] Player " .. pname .. " becomes a wizard")
 	end,
 	on_unassigned = function(pname)
 		mana.setmax(pname, mana.getmax(pname)-100)
 	end,
+	on_update = function(pname)
+		local staminavalue = 30
```
Comment (Darcidride) [inline mods/mff/mff_pclasses/wizard.lua]: should be 10 stamina for Wizard, no ?

**Comment by Darcidride** [comment]
I was a small error with the Wizard stamina, i let you fix the review, then :+1: For me

Thank you & good work :)

File: mods/mff/mff_pclasses/wizard.lua (hunk 1 above)
Comment (crabman77) [inline mods/mff/mff_pclasses/wizard.lua]: it's a typo, fixed in 2nd commit.

Comment (Darcidride) [inline mods/mff/mff_pclasses/wizard.lua]: Damn, my fault, yes i saw it :) :+1:

**Comment by LeMagnesium** [comment]
OK by me. Merging.

**Comment by LeMagnesium** [comment]
https://github.com/MinetestForFun/server-minetestforfun/commit/ee58c4930fa75f0bbaeb8a8b3374855bba9fd625
https://github.com/MinetestForFun/server-minetestforfun/commit/126ce0d7e421307cb2eb35eccf78bc622aeca268
Working on further improvements.

**Pull request closed without merging**